clients/drcachesim/tools/basic_counts.cpp

```diff
@@ -106,6 +113,13 @@ basic_counts_t::parallel_shard_memref(void *shard_data, const memref_t &memref)
             ++counters->encodings;
     } else if (memref.instr.type == TRACE_TYPE_INSTR_NO_FETCH) {
         ++counters->instrs_nofetch;
+        if (TESTANY(OFFLINE_FILE_TYPE_SYSCALL_PT, per_shard->filetype_)) {
```

Comment:
Matching what we do currently, we should either add a new counter for these kernel_nofetch instrs, or just not count them. I think the latter option is okay because the kernel won't have these no-fetch instrs.

Reply:
Done.